i18n/config.json

```diff
@@ -0,0 +1,370 @@
+[
+  {
+    "code": "en",
+    "hrefLang": "en",
+    "name": "English",
+    "localName": "English",
+    "langDir": "ltr",
```

Comment:
Doesn't appear that we actually use `langDir` for anything - should we?

Reply:
I know we use the `isLangRightToLeft` helper for that currently.

Reply:
Exactly. Related to prev comment. I haven't done that yet because I didn't want to add too many changes but I think that for `langDir` we should use it from here as the source of truth.